endpoints/openrtb2/video_auction.go

```diff
@@ -617,7 +618,9 @@ func (deps *endpointDeps) parseVideoRequest(request []byte, headers http.Header)
 
 	//if Device.UA is not present in request body, init it with user-agent from request header if it's present
 	if req.Device.UA == "" {
-		req.Device.UA = headers.Get("User-Agent")
+		ua := headers.Get("User-Agent")
+		ua, _ = url.QueryUnescape(ua)
```

Comment:
This is a change which affects everyone. How will this behave when the user agent is not url encoded? Also, please never ignore errors.

Reply:
fyi we tested with a value that's not encoded and QueryUnescape() returns the same value in that case. In the unlikely event that an unencoded value contains a decodable character sequence the return wouldn't match, but that's probably a pretty extreme edge case.

Reply:
Yes, fixed

Reply:
Looking at samples of UA strings, the '+' character may sometimes appear and the '%' character really doesn't. The `+` character would get changed to a space if not actually encoded. Perhaps you can check for the presence of a '%' as a best guess if the UA is encoded? Whatever you think is the best choice for this usecase. Please document your decision / explanation in a code block and also in the docs. This is one of those small hidden things which may infuriate someone down the road trying to debug.

GoLang already checks for invalid escape sequences and errors out, which now causes you to fallback to the original string.. which is good.

Reply:
Good point! Check added. 
I also added a comment in code. Do you want me to mention it in prebid.org docs?

Reply:
Comment is good for this PR. Thank you. I would like it to be mention in prebid.org docs as well, but I see that as a follow-up and not a blocker.